### ISSUE TITLE
Rogue tab group is off the screen -- can't close it or move it

### ISSUE THREAD

**Issue by nathan-b** [projectdelphai/panorama-tab-groups#160]
I have five tab groups. Four of them are arranged in a grid and I can see them nicely. The fifth one is only about half visible, and the top half is off the top of the screen. This means that I can't reposition it or close it.

There would be three ways that this issue could be fixed to my satisfaction (that I can think of in the moment):
1. Make it so that the top of a tab group window can't be moved off the edge of the screen, and any window that is off the edge of the screen gets moved until its controls are visible.
2. Make it so I can drag tab group windows by clicking and dragging anywhere in the window (or maybe just anywhere that's not covered by an actual tab icon)
3. Provide a dropdown list of all tab groups with the option to close the group from the list. I don't actually care about the group or any tabs that might be in it (I don't even see any tabs in it, so it may be empty).

**Comment by sleepindawg** [comment]
@nathan-b 
Try zooming in to 300% and see if there are any tabs hidden offscreen, I had the same problem but had several rows of tabs in the group including a row that was hidden. I halfway fixed the problem by opening a tab in the group and then hauling all the tabs off the browser so they became their own windows. Once I got them all off the browser window I opened a group that I could see all of and dropped one in it. From there it was just a matter of creating a new group with that one tab and dropping the rest of the tabs in that group.

All of this left me with just the blank tab in the group and it wasn't visible until I zoomed in to 300%. I can't get rid of what's left of the rogue group window, but at least I got all my good tabs back from it.

BTW, for me, this problem started with a new tab group which turned up malformed with the bottom off the screen and I was unable to resize it, later when I opened the groups view I found the top of the rogue group off the top of the screen. This paragraph is for the devs, this info may help them find the source of the problem.
